TestTools/StreamChatTestTools/Extensions/XCTest+Helpers.swift

```diff
@@ -23,6 +23,14 @@ public func XCTAssertEqual<T: Equatable>(_ expected: T,
     }
 }
 
+/// Asserts diff between the expected and received collections after sorting both collections
+public func XCTAssertEqualIgnoringOrder<T: RandomAccessCollection>(_ expected: T,
```

Comment:
Why do we need this? Shouldn't the order of the collections be the same?

Reply:
I can remove it and add more pre-defined dummy data which makes sure the final result is correctly sorted